<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyCart</class>
 <widget class="QWidget" name="MyCart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyCart</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="table">
     <column/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="checkout_button">
       <property name="text">
        <string>ชำระเงิน</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="printButton">
       <property name="text">
        <string>พิมพ์ใบเสร็จ</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
